alert-database/src/main/java/com/synopsys/integration/alert/database/api/StaticJobAccessorV2.java

```diff
@@ -193,17 +207,12 @@ public DistributionJobModel updateJob(UUID jobId, DistributionJobRequestModel re
 
     @Override
     @Transactional
-    public void deleteJob(UUID jobId) {
-        distributionJobRepository.deleteById(jobId);
-    }
+    public void deleteJob(UUID jobId) throws AlertConfigurationException {
+        if (jobId == null) {
+            throw new AlertConfigurationException("Cannot delete job with null id.");
+        }
```

Comment:
This may have gotten lost in translation, but I think we should skip the null check. Accessors are "internal" APIs, so if the values they receive are null, then it's a bug. I think we decided than anything without a `@Nullable` annotation should be assumed not to be null except when it comes to objects received in REST requests.

Reply:
Sorry! I apparently read the opposite of what you said. I've removed the null checks and the associated tests.